osu.Game/Overlays/ModSelection/ModSelection.cs

```diff
@@ -0,0 +1,520 @@
+                ScoreMultiplier = 0.5
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.HalfTime,
+                Icon = FontAwesome.fa_osu_mod_halftime,
+                ScoreMultiplier = 0.3
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.HardRock,
+                Icon = FontAwesome.fa_osu_mod_hardrock,
+                ScoreMultiplier = 1.06 // FIXME
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.SuddenDeath,
+                Icon = FontAwesome.fa_osu_mod_suddendeath,
+                ScoreMultiplier = 1.0
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Perfect,
+                Icon = FontAwesome.fa_osu_mod_perfect,
+                ScoreMultiplier = 1.0
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.DoubleTime,
+                Icon = FontAwesome.fa_osu_mod_doubletime,
+                ScoreMultiplier = 1.12 // FIXME
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Nightcore,
+                Icon = FontAwesome.fa_osu_mod_nightcore,
+                ScoreMultiplier = 1.12 // FIXME
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Hidden,
+                Icon = FontAwesome.fa_osu_mod_hidden,
+                ScoreMultiplier = 1.06 // FIXME
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Flashlight,
+                Icon = FontAwesome.fa_osu_mod_flashlight,
+                ScoreMultiplier = 1.12 // FIXME
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Relax,
+                Icon = FontAwesome.fa_osu_mod_relax,
+                ScoreMultiplier = 0
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Relax2,
+                Icon = FontAwesome.fa_osu_mod_autopilot,
+                ScoreMultiplier = 0
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Target,
+                Icon = FontAwesome.fa_osu_mod_target,
+                ScoreMultiplier = 0
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.SpunOut,
+                Icon = FontAwesome.fa_osu_mod_spunout,
+                ScoreMultiplier = 0
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Autoplay,
+                Icon = FontAwesome.fa_osu_mod_auto,
+                ScoreMultiplier = 0
+            });
+            TempMods.Add(new Mod
+            {
+                Name = Mods.Cinema,
+                Icon = FontAwesome.fa_osu_mod_cinema,
+                ScoreMultiplier = 0
+            });
+        }
+        // ] TEMP: remove when proper mod system will be added

```

Comment:
just add a `//TODO:` comment at the top of the method, rather than doing this weird comment backets stuff.